packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -74,8 +74,8 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
 
     return (
       <div className={ css('ms-Breadcrumb', className, styles.root) } ref='renderingArea'>
-        <FocusZone direction={ FocusZoneDirection.horizontal }>
-          <ul className={ css('ms-Breadcrumb-list', styles.list) }>
+        <FocusZone direction={ FocusZoneDirection.horizontal } role='navigation'>
```

Comment:
@cschleiden I think you are referring to the "main", "banner", and "contentinfo" landmark roles; there should only be at most one element with each of those roles. Navigation sound like the correct role to use here